### PR TITLE
changelog: clarify 1.17 upgrade requirements

### PR DESCRIPTION
**Pull request by bhcleek**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Clarify the 1.17 upgrade requirements for storage so that it doesn't
ready as if an entire cluster has to be drained before any node can be
upgraded.

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Comment by saschagrunert** [comment]
Seeking for approval by @release-engineering

File: CHANGELOG-1.17.md
Comment (msau42) [inline CHANGELOG-1.17.md]: This can be simplified to:

"A node that uses a CSI raw block volume needs to be drained before kubelet can be upgraded to 1.17"

Comment (bhcleek) [reply CHANGELOG-1.17.md]: PTAL

**Comment by msau42** [comment]
/lgtm

Should we cherry pick this to release-1.17 branch as well?

**Comment by MorrisLaw** [comment]
Flaky tests? 🤔 

hey @alejandrox1 👋 , what do we need to do to get this in?

**Comment by saschagrunert** [comment]
@MorrisLaw do you want to update the website as well or should I take care of that?

**Comment by MorrisLaw** [comment]
Either or is fine. I don't mind doing it, I just haven't updated the website before, is it just making a PR with the new verbiage, to the project that website is in @saschagrunert ?

**Comment by saschagrunert** [comment]
> Either or is fine. I don't mind doing it, I just haven't updated the website before. Is it just making a PR with the new verbiage, to the project that website is in @saschagrunert ?

Yes, it’s just updating the markdown release notes in https://github.com/kubernetes/website 🙂

**Comment by MorrisLaw** [comment]
Awesome! Should I wait until this is merged or make the PR and note that we should merge pending these changes? @saschagrunert

**Comment by saschagrunert** [comment]
> Awesome! Should I wait until this is merged or make the PR and note that we should merge pending these changes? @saschagrunert

I think both would be okay since this PR is already on its way into the master :)

**Comment by bhcleek** [comment]
I thought that https://github.com/kubernetes-sigs/release-notes/pull/137 was the website update based on https://kubernetes.slack.com/archives/C2C40FMNF/p1578682428029300?thread_ts=1578677838.025500&cid=C2C40FMNF

**Comment by MorrisLaw** [comment]
> I thought that [kubernetes-sigs/release-notes#137](https://github.com/kubernetes-sigs/release-notes/pull/137) was the website update based on https://kubernetes.slack.com/archives/C2C40FMNF/p1578682428029300?thread_ts=1578677838.025500&cid=C2C40FMNF

**TLDR;** one updates https://relnotes.k8s.io and the other updates  https://kubernetes.io

Good question @bhcleek , I wasn't sure about this when you brought this up, but looking through the two projects and based on the PRs it looks like:

The PR I did: [kubernetes/website](https://github.com/kubernetes/website/pull/18921), points to https://kubernetes.io/docs/setup/release/notes/#storage-1.
Your PR: [kubernetes-sigs/release-notes](https://github.com/kubernetes-sigs/release-notes/pull/137), points to somewhere in https://relnotes.k8s.io/

Is this correct @saschagrunert ?

**Comment by saschagrunert** [comment]
Yep you’re right @MorrisLaw :)

**Pull request merged**
